resources/build/help-keyman-com.sh

```diff
@@ -0,0 +1,202 @@
+
+if [ -t 2 ]; then
+  t_red=$'\e[1;31m'
+  t_grn=$'\e[1;32m'
+  t_yel=$'\e[1;33m'
+  t_blu=$'\e[1;34m'
+  t_mag=$'\e[1;35m'
+  t_cyn=$'\e[1;36m'
+  t_end=$'\e[0m'
+fi
+
+#
+# Uploading Keyman documentation
+#
+
+##
+## Upload documentation updates to help.keyman.com
+## Paths depend on $platform
+##
+function upload_keyman_help {
+
+  local helppath
+  local dstpath
+
+  case $platform in
+    android)
+      helppath=$KEYMAN_ROOT/android/help
+      dstpath="$HELP_KEYMAN_COM/products/android/$VERSION_RELEASE"
+      ;;
+    ios)
+      helppath=$KEYMAN_ROOT/ios/help
+      dstpath="$HELP_KEYMAN_COM/products/iphone-and-ipad/$VERSION_RELEASE"
+      ;;
+    linux)
+      helppath=$KEYMAN_ROOT/linux/help
+      dstpath="$HELP_KEYMAN_COM/products/linux/$VERSION_RELEASE"
+      ;;
+    mac)
+      helppath=$KEYMAN_ROOT/mac/help
+      dstpath="$HELP_KEYMAN_COM/products/mac/$VERSION_RELEASE"
+      ;;
+    windows)
+      # Note: `/windows/src/desktop/help/build.sh web` must be run first
+      helppath=$KEYMAN_ROOT/windows/bin/help/md/desktop
+      dstpath="$HELP_KEYMAN_COM/products/windows/$VERSION_RELEASE"
+      ;;
+    *)
+      display_usage
+    esac
+
+  #
+  # Look for help source folder.
+  #
+
+  if [[ ! -d "$helppath" ]]; then
+    echo "${t_yel}Warning: The source path $helppath does not exist${t_end}"
+    exit 1
+  fi
+
+  mkdir -p "$dstpath"
+
+  rm -rf "$dstpath/*"
+  cp -r "$helppath"/* "$dstpath/"
+}
+
+#
+# Commit and push to the help.keyman.com repo
+# Creates a pull request with the 'auto' label
+# Which a GitHub action will watch for in order
+# to automatically process and merge it
+#
+
+function commit_and_push {
+  echo "Committing and pushing updated Keyman for $platform documentation"
+
+  pushd $HELP_KEYMAN_COM
+
+  if [ ! -z "${TEAMCITY_VERSION-}" ]; then
+    git config user.name "Keyman Build Server"
+    git config user.email "keyman-server@users.noreply.github.com"
+  fi
+
+  local branchname="auto/$platform-help-$VERSION_WITH_TAG"
+  local modifiedfiles="$HELP_KEYMAN_COM/products/$platform/$VERSION_RELEASE"
+
+  # Base branch depends on the tier
+  local basebranch="master"
+  if [ "$TIER" == "alpha" ] || [ "$TIER" == "beta" ]; then
+      basebranch="staging"
+  fi
+
+  git checkout -b $branchname $basebranch
+  git add $modifiedfiles || return 1
+  git diff --cached --no-ext-diff --quiet --exit-code && {
+    # if no changes then don't do anything.
+    echo "No changes to commit"
+    popd
+    return 0
+  }
+
+  echo "changes added to cache...>>>"
+  local commitmessage="auto: Keyman for $platform help deployment"
+  git commit -m "$commitmessage" || return 1
+  git push origin $branchname || return 1
+  hub pull-request -b $basebranch -l auto -m "$commitmessage" || return 1
+  popd
+
+  echo "Push to help.keyman.com complete"
+
+  return 0
+}
+
+#
+# Main
+#
+
+platform=
+
+# Process command-line arguments
+while [[ $# -gt 0 ]] ; do
+  key="$1"
+  case $key in
+    -help|-h)
+      display_usage
+      exit
```

Comment:
```suggestion
      exit 0
```

It's not an error, I guess...